src/lib/utils/time.ts

```diff
@@ -7,12 +7,20 @@ export const formatSecondsToDuration = (seconds: number) => {
   const hours = duration.asHours()
   const days = duration.asDays()
   const months = duration.asMonths()
+  const minutes = duration.asMinutes()
 
+  if (seconds < 60) {
+    return `${Math.floor(seconds)} second${Math.floor(seconds) !== 1 ? 's' : ''}`
+  }
+
+  if (minutes < 60) {
+    return `${Math.floor(minutes)} minute${Math.floor(minutes) !== 1 ? 's' : ''}`
+  }
   if (hours < 24) {
     return `${Math.floor(hours)} hour${Math.floor(hours) !== 1 ? 's' : ''}`
-  } else if (days < 31) {
+  }
+  if (days < 30) {
```

Comment:
was this inteded to be changed? 30 days now 1 month?

Reply:
It was! Noticed it was weird while introducing tests. Happy to change it back if we prefer the previous flavour.

Reply:
I don't know how much of an opinion, just wanted to clarify. Maybe @shapeshift/product would care or have an opinion

Reply:
Personally I think 1 month might get lost vs 30 days. @twblack88

Reply:
agree with @reallybeard I vote days because : 
- more scalable & clear for edges around epochs 187 days versus 3... yadda yadda months
- more holistic elsewhere in codebase
- some psycho in governance will change epochs to be smaller. so we can adjust on the fly

Reply:
talked through with @twblack88 and let's have days go up to 30 for the average "standard" max day display

Reply:
@reallybeard @twblack88 FYI, @firebomb1 has proposed a great solution that will both keep things consistent *and* localized: https://day.js.org/docs/en/durations/locale

Moving this to draft to use this instead of hand-rolled solution tomorrow!

Reply:
🔥 💣  FTW

Reply:
https://github.com/shapeshift/web/pull/7016/commits/70877f3dcbec29889df039a48ca2de35f86f9cf8 note this uses default of dayjs - tried to play with [thresholds](https://day.js.org/docs/en/customization/relative-time) for an hour or so but rage quitted, made things quite borked

Reply:
Yes, and then these thresholds would also apply everywhere in the app once customized. We'd also need to customize them for each language (i.e. get translations for all the languages we support and maintain this each time a new language is added/removed). It's preferable if we can avoid this in my opinion. Thanks for the fix!